subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -659,6 +663,16 @@ int nrf_cloud_decode_data_endpoint(const struct nrf_cloud_data *input,
 		return err;
 	}
 
+	/* Populate batch endpoint topic by copying and appending /batch to the parsed
+	 * tx endpoint (d2c) topic.
+	 */
+	size_t batch_ep_len_temp = tx_endpoint->len + sizeof(JSON_KEY_TOPIC_BATCH);
+
+	batch_endpoint->ptr = k_malloc(batch_ep_len_temp);
```

Comment:
you could use `nrf_cloud_calloc` just to be consistent with the rest of the lib.

Reply:
Will do!